src/test/java/hudson/plugins/jobConfigHistory/FileHistoryDaoTest.java

```diff
@@ -385,7 +385,9 @@ public void testGetSystemRevisionAmount_Single() {
 //        System.out.println("jenkinsloc...=" + getJenkinsRuleSut().getSystemHistory("jenkins.model.JenkinsLocationConfiguration").size());;
         assertEquals(3, getJenkinsRuleSut().getSystemRevisionAmount("config"));
         assertEquals(1, getJenkinsRuleSut().getSystemRevisionAmount("jenkins.telemetry.Correlator"));

```

Comment:
One could also add an assertion for `jenkins.model.JenkinsLocationConfiguration` after lne 386. Then all three currently generated histories for system configuration are covered.

Reply:
done